### PR TITLE
Initial models

### PR DESCRIPTION
**Pull request by llagong**
This PR includes:

- 2 new models based on the TOR.

### CODE REVIEW

**Comment by llagong** [comment]
Complete PR #1 first then apply changes.

File: src/schemas/db/node-contacts.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+module.exports = connection => {
+  const NodeContactSchema = new Schema(
+    {
+      // Handling of different types of contact
+      type: {
+        type: Schema.Types.String,
+        enum: ['direct-bluethooth', 'direct-network', 'indirect', 'manual'],
```
Comment (jananxd) [inline src/schemas/db/node-contacts.js]: I think this is a typo `direct-bluethooth`.

Comment (llagong) [reply src/schemas/db/node-contacts.js]: Didn't notice. Will fix it right away. Thanks!

File: src/models/nodes.model.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const NodeSchema = new Schema(
+  {
+    // As per TOR, this value must be determined based on device_id + person_id.
+    node_id: { type: Schema.Types.String, required: true },
+
+    device_id: { type: Schema.Types.String, required: true },
```
Comment (kvdomingo) [inline src/models/nodes.model.js]:
```suggestion
    device_id: { 
        type: [Schema.Types.String],
        required: true
    },
```

**Pull request merged**